src/main/java/macrobase/runtime/command/MacroBasePipelineCommand.java

```diff
@@ -1,28 +1,37 @@
-package macrobase.runtime.standalone;
+package macrobase.runtime.command;
 
 import io.dropwizard.cli.ConfiguredCommand;
 import io.dropwizard.setup.Bootstrap;
 import macrobase.MacroBase;
 import macrobase.analysis.pipeline.BasicBatchedPipeline;
+import macrobase.analysis.pipeline.Pipeline;
 import macrobase.analysis.result.AnalysisResult;
 import macrobase.conf.MacroBaseConf;
 import net.sourceforge.argparse4j.inf.Namespace;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class MacroBaseBatchCommand extends ConfiguredCommand<MacroBaseConf> {
-    private static final Logger log = LoggerFactory.getLogger(MacroBaseBatchCommand.class);
+public class MacroBasePipelineCommand extends ConfiguredCommand<MacroBaseConf> {
+    private static final Logger log = LoggerFactory.getLogger(MacroBasePipelineCommand.class);
 
-    public MacroBaseBatchCommand() {
-        super("batch", "Run task without starting server.");
+    public MacroBasePipelineCommand() {
+        super("pipeline", "Run pipeline without starting server.");
     }
 
     @Override
     protected void run(Bootstrap<MacroBaseConf> bootstrap,
                        Namespace namespace,
                        MacroBaseConf configuration) throws Exception {
         configuration.loadSystemProperties();
-        BasicBatchedPipeline analyzer = new BasicBatchedPipeline(configuration);
+        Class c = Class.forName(configuration.getString(MacroBaseConf.PIPELINE_NAME));
+        Object ao = c.newInstance();
```

Comment:
If we don't want to do this initialize stuff, does using getConstructor as described here work? http://stackoverflow.com/a/6094600